budou/budou.py

```diff
@@ -75,13 +78,19 @@ def main():
       inlinestyle=args['--inlinestyle'],
       wbr=args['--wbr'],
       )
-  html_code = result['html_code']
-  if not isinstance(html_code, str):
-    html_code = html_code.encode('utf-8')
-  print(html_code)
+
+  if args['--separator']:
+    output = result['chunks'].separator_serialize(args['--separator'])
+  else:
+    output = result['html_code']
+    
+  if not isinstance(output, str):
+    output = output.encode('utf-8')
+  print(output)
+
   sys.exit()
 
-def parse(source, segmenter='nlapi', language=None, max_length=None,
+def parse(source, segmenter='nlapi', language=None, max_length=None, separator=None,
```

Comment:
Could you please `separator=None` to the last in the arguments (i.e. next to `wbr=False`) to keep the backward compatibility for users who are using this method without keyword arguments?

Reply:
oops, shouldn't even be there